blog_api/lib/blog_api.ex

```diff
@@ -4,4 +4,6 @@ defmodule BlogApi do
   defdelegate create_user(params), to: User.Create, as: :call
 
   defdelegate fetch_user(id), to: User.Get, as: :call
+
+  defdelegate list_users(), to: User.Get, as: :list_users
```

Comment:
Pelo oque entendi, você está estruturando o projeto em uma pega mais de _use cases_ neh? Se for assim imagino que o`list_users` deveria ficar em outro arquivo -> `BlogApi.User.List`. Faz sentido?